packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -531,26 +566,142 @@ class CupertinoPopupSurface extends StatelessWidget {
   /// Some popups, like iOS's volume control popup, choose to render a blurred
   /// area without any white paint covering it. To achieve this effect,
   /// [isSurfacePainted] should be set to false.
+  ///
+  /// Defaults to true.
   final bool isSurfacePainted;
 
   /// The widget below this widget in the tree.
-  final Widget? child;
+  ///
+  /// Because [CupertinoPopupSurface] is composed of proxy boxes, which mimic
+  /// the size of their child, a [child] is required to ensure that this surface
+  /// has a size.
+  final Widget child;
+
+  /// The default strength of the blur applied to widgets underlying a
+  /// [CupertinoPopupSurface].
+  static const double defaultBlurSigma = 30.0;
+
+  /// The default corner radius of a [CupertinoPopupSurface].
+  static const BorderRadius _clipper = BorderRadius.all(Radius.circular(14));
+
+  // The ColorFilter matrix used to saturate widgets underlying a
```

Comment:
uber nit: consider putting symbols such as `ColorFilter` in square brackets, they won't show up in public API docs, but the analyzer will analyze / link them.

Reply:
Done! Good to know.